src/main/java/edu/wpi/first/pathweaver/Path.java

```diff
@@ -44,7 +45,7 @@ public String toString() {
   }
 
   /**
-   * Creates new Waypoint in Path after previous.
+   * Create new Waypoint in Path after previous.
```

Comment:
Creates

Reply:
Whoops, chose the wrong line in a merge conflict. Should be fixed now.